src/libmongoc/tests/json-test.c

```diff
@@ -1693,6 +1695,23 @@ run_json_general_test (const json_test_config_t *config)
          continue;
       }
 
+      if (config->skips) {
+         test_skip_t *iter;
+         bool should_skip = false;
+
+         for (iter = config->skips; iter->description != NULL; iter++) {
```

Comment:
Tantalizingly close to a general find() on associative array keys, but here we have the complication of the special field names.

Reply:
I am not sure I understand. Are you referring to an associative array type in C?